### PR TITLE
NO-ISSUE: Add event severity documentation

### PR DESCRIPTION
**Pull request by arielireni**
Added documentation regarding events severity levels.

Signed-off-by: arielireni <aireni@redhat.com>

# Assisted Pull Request

## List all the issues related to this PR

- [ ] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Review by masayag — APPROVED** [review]

File: docs/dev/events.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ cluster_id
    1. "cluster" type requires the existence of "cluster_id" in properties.
    2. "host" type requires the existence of "host_id" and "infra_env_id" in properties.
    3. "infra_env" type requires the existence of "infra_env_id" in properties.
-4. __severity__: Any of "info", "warning", "error" or "critical".
+4. __severity__: Any of "info", "warning", "error" or "critical". See more info about severity levels [here](https://github.com/openshift/assisted-service/blob/master/docs/events.md).
```
Comment (nmagnezi) [inline docs/dev/events.md:28]:
```suggestion
4. __severity__: Any of "info", "warning", "error" or "critical". See more info about severity levels [here](../events.md).
```

**Review by nmagnezi — CHANGES_REQUESTED** [review]
Please use a short path link instead.
This will play nicer with branches along the way. meaning this doc, when it becomes a part of  the ocm-2.5 branch, will point to the ocm-2.5 events.md

File: docs/dev/events.md (hunk 1 above)
Comment (arielireni) [reply docs/dev/events.md]: done

File: docs/dev/events.md [hunk 2]
```diff
@@ -26,7 +26,14 @@ cluster {cluster_id}"` expects the existence of a property named
    1. "cluster" type requires the existence of `cluster_id` in properties.
    2. "host" type requires the existence of `host_id` and `infra_env_id` in properties.
    3. "infra_env" type requires the existence of `infra_env_id` in properties.
+<<<<<<< HEAD
```
Comment (nmagnezi) [inline docs/dev/events.md:29]: seems like there was an issue with the rebase

**Review by nmagnezi — CHANGES_REQUESTED** [review]
thanks for addressing my comment.
There seems to be an issue with the rebase. Fix it and I'll approve the PR.

**Review by nmagnezi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**